.env.example

```
@@ -219,20 +237,20 @@ CHANGEDETECTION_MINIMUM_SECONDS_RECHECK_TIME=60
 
 # Webhook secret for changedetection.io -> webhook bridge (generate with: openssl rand -hex 32)
 # ⚠️  Development default below is UNSAFE for production - generate a new secret!
+# Note: WEBHOOK_CHANGEDETECTION_HMAC_SECRET uses substitution to match this value
 CHANGEDETECTION_WEBHOOK_SECRET=<64-char-hex>
 # changedetection.io API key (get from Settings > API in changedetection.io UI)
 CHANGEDETECTION_API_KEY=<changedetection-api-key>
```

dotenv-linter (4.0.0)

[warning] [UnorderedKey] The CHANGEDETECTION_API_KEY key should go before the CHANGEDETECTION_WEBHOOK_SECRET key

(UnorderedKey)